admin/src/main/resources/io/buoyant/admin/js/src/dtab_viewer.js

```diff
@@ -20,6 +22,7 @@ define(['jQuery'], function($) {
         return { prefix: tuple[0], dst: tuple[1] };
       });
       this.render();
+      $(window).trigger("resize");
```

Comment:
I'm a little confused here since triggering `resize` invokes `render`, but `render` is already invoked in the previous line...